Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -92,7 +92,7 @@ below. This will ensure all pods are managed by Cilium.
       --set global.cni.binPath=/home/kubernetes/bin \\
       --set global.gke.enabled=true \\
       --set config.ipam=kubernetes \\
-      --set global.native-routing-cidr=$NATIVE_CIDR
+      --set global.nativeRoutingCIDR=$NATIVE_CIDR
```

Comment:
Note that we need this particular change on v1.7 as well (but not the ipam option). Given you're on backports this week, probably easiest for you to make sure the right commit from this PR goes back.

Reply:
Actually, it looks like we backported this change to the docs but never backported the helm side interpretation so it has been a no-op the entire time :facepalm: